src/clientSideScene/CameraControls.ts

```diff
@@ -233,12 +234,14 @@ export class CameraControls {
     this.target = new Vector3()
     this.domElement = domElement
     this.isDragging = false
+    this.wasDragging = false
     this.mouseDownPosition = new Vector2()
     this.mouseNewPosition = new Vector2()
 
     this.domElement.addEventListener('pointerdown', this.onMouseDown)
     this.domElement.addEventListener('pointermove', this.onMouseMove)
     this.domElement.addEventListener('pointerup', this.onMouseUp)
+    this.domElement.addEventListener('click', this.onClick)
```

Comment:
We haven't used the [click](https://developer.mozilla.org/en-US/docs/Web/API/Element/click_event) event before for camera controls. It only fires if the element is the same as where mousedown was. If you're not able to get the sketch fix working, I might try switching to the pointerup event handler. But I think it's okay since we use `setPointerCapture()` to ensure that it is the same element.

Reply:
definitely feel free to take this over.  my approach here isn't with the same confidence that my previous PR had.

Reply:
Yeah no sweat we'll slot this into our existing `onMouseDown` listener, I think I get what you're doing here